data/hai/hai reveal 1 intro.txt

```diff
@@ -11,48 +11,776 @@
+					accept
+				`	(Loiter to find out more.)`
+					goto loiter
+				`	"What were we getting paid for this?"`
+			`	He frowns. "It was based on your ship's combat quality. I don't have that tablet with me right now though. Once the supply ships have arrived I should have access to that again. Now go on, you won't get paid at all if the supply ships don't make it."`
+				accept
+			label loiter
+			`	He blinks at you. "Well, what are you waiting for? Go! Come back when all the supply ships have arrived safely."`
+			`	You are left with little choice other than to nod and get on your way.`
+				accept
+	on accept
+		set "hr: sandsnap project"
+	on visit
+		dialog "The supply ships have not all arrived safely yet."
+	on fail
+		dialog "The supply convoy has been lost."
+	npc accompany
+		government "Hai Merchant"
+		personality escort timid
+		system "Ula Mon"
+		fleet "Large Hai Merchant" 3
+	npc
+		government "Pirate"
+		personality coward opportunistic vindictive
+		system "Waypoint"
+		fleet "Marauder fleet IX" 2
+	on complete
+		"reputation: Hai" += 200
+		"reputation: Hai (Wormhole Access)" += 200
+		"reputation: Hai Merchant" += 200
+		"reputation: Hai Merchant (Human)" += 200
+		"reputation: Hai Merchant (Sympathizers)" += 200
+		event "hr: sandsnap awoken" 5
+		fail "HRCP: Sandsnap Mercenaries"
+		# Ideally eventually use: payment ( "armament deterrence" * 18000 - "cargo attractiveness" * 3000 )
+		payment 1200000
+		conversation
+			`As you're re-docking behind the supply convoy, a message pings on your console. It contains a short text from "Heyru" thanking you for your assistance and advising that you've been sent a payment of <payment> for your efforts.`
+			`	It seems that the security mechanisms in the docking section have been reactivated, so they no longer need their mercenaries to hang around.`
+
 
+event "hr: sandsnap awoken"
+	planet Sandsnap
+		government "Hai Merchant"
+		attributes hai station
+		landscape land/station35
+		music ambient/machinery
+		description `This station was recently re-powered by a private military contractor to serve as a resupply and repair base that Hai warships can be stationed at to respond more rapidly to pirates entering the system. There has been a constant presence here since it was set up shortly after the pirate raids on Allhome.`
+		spaceport `The operating model for the spaceport is evidently based on the military base at Everhope, but the station layout makes restricting portions of the promenade unfeasible. This leaves the dual-level promenade fully open to civilian access, with security restrictions in place at all access points to the higher levels. The floor has had new texturing laid down to prevent slipping, but nearly every other surface appears to be coated in a fine, sticky, metallic powder. Frequently touched surfaces have been polished clean, but any accidental contact with the powder results in leaving fingerprints on any non-metallic surface until thoroughly wiped off.`
```

Comment:
```suggestion
		spaceport `The operating model for the spaceport is evidently based on the military base at Everhope, but Sandsnap's layout makes restricting portions of the promenade unfeasible. While this leaves the dual-level promenade fully open to civilian access, security restrictions have been put in place at all access points to the higher levels.`
		`	The floor has had new texturing laid down to prevent slipping, but nearly every other surface appears to be coated in a fine, sticky, metallic powder. Frequently touched surfaces have been polished clean, but otherwise any accidental contact with the powder results in fingerprints being left on any non-metallic surface until it is thoroughly wiped off.`
```
I understand if changing `station layout` to `Sandsnap's layout` might not be preferable since it's about civilian station layouts, but `Sandsnap's layout` is more clear to me.